README.md

```diff
@@ -1,49 +1,61 @@
+  &nbsp;&nbsp;&bull;&nbsp;&nbsp;
+  <a href="https://martinvonz.github.io/jj/v0.9.0/tutorial.html">Getting Started
+  Tutorial</a>
+</strong>
 
-This is not a Google product. It is an experimental version-control system
-(VCS). I (Martin von Zweigbergk <martinvonz@google.com>) started it as a hobby
-project in late 2019. That said, this is now my full-time project at Google. My
-presentation from Git Merge 2022 has information about Google's plans. See the
-[slides](https://docs.google.com/presentation/d/1F8j9_UOOSGUN9MvHxPZX_L4bQ9NMcYOp1isn17kTC_M/view)
-or the [recording](https://www.youtube.com/watch?v=bx_LGilOuE4).
+</div>
 
 ## Introduction
 
-Jujutsu is a
-[Git-compatible](https://martinvonz.github.io/jj/latest/git-compatibility)
-[DVCS](https://en.wikipedia.org/wiki/Distributed_version_control). It combines
+Jujutsu is a [Distributed Version Control System](https://en.wikipedia.org/wiki/Distributed_version_control) that is [Git-compatible](https://martinvonz.github.io/jj/latest/git-compatibility). It combines
 features from Git (data model,
 [speed](https://github.com/martinvonz/jj/discussions/49)), Mercurial (anonymous
-branching, simple CLI [free from "the
-index"](https://martinvonz.github.io/jj/latest/git-comparison#the-index),
+branching, simple CLI
+[free from "the index"](https://martinvonz.github.io/jj/latest/git-comparison#the-index),
 [revsets](https://martinvonz.github.io/jj/latest/revsets), powerful
-history-rewriting), and Pijul/Darcs ([first-class
-conflicts](https://martinvonz.github.io/jj/latest/conflicts)), with features not
-found in most of them
+history-rewriting), and Pijul/Darcs
+([first-class conflicts](https://martinvonz.github.io/jj/latest/conflicts)),
+with features not found in most of them
 ([working-copy-as-a-commit](https://martinvonz.github.io/jj/latest/working-copy),
 [undo functionality](https://martinvonz.github.io/jj/latest/operation-log),
-automatic rebase, [safe replication via `rsync`, Dropbox, or distributed file
+automatic rebase,
+[safe replication via `rsync`, Dropbox, or distributed file
 system](https://martinvonz.github.io/jj/latest/technical/concurrency)).
 
 The command-line tool is called `jj` for now because it's easy to type and easy
 to replace (rare in English). The project is called "Jujutsu" because it matches
 "jj".
 
-If you have any questions, please join us on Discord
+Jujutsu is relatively young, with lots of work to still be done. If you have any
+questions, want to solicit advice, or talk about future plans, please join us on
+Discord
 [![Discord](https://img.shields.io/discord/968932220549103686.svg?label=&logo=discord&logoColor=ffffff&color=7389D8&labelColor=6A7EC2)](https://discord.gg/dkmfj3aGQN)
-or start a [GitHub Discussion](https://github.com/martinvonz/jj/discussions).
+or start a [GitHub Discussion](https://github.com/martinvonz/jj/discussions)
+&mdash; the developers monitor both channels.
```

Comment:
Minor nit: I think it might be better to replace this long dash with a period or a semicolon.